kie-user-bom-parent/pom.xml

```diff
@@ -74,6 +88,33 @@
       </snapshots>
     </repository>
   </repositories>
```

Comment:
If we keep this ugly hack, there needs to be a comment in the pom to explain why. We'll probably want to explicitly mention that this is make from sources builds succeed immediately after a release when Maven Central hasn't synced yet.